src/createBot.test.ts

```diff
@@ -191,17 +193,24 @@ describe('send functions', () => {
   });
 });
 
+function log(line: string) {
+  fs.appendFileSync('./log.txt', `${line}\n`);
```

Comment:
Nit: can we have the logfile in a tmp directory? e.g. `{APP_ROOT}/tmp/log.txt`. Then ignore the entire `tmp` folder